flutter-studio/src/io/flutter/profiler/FlutterStudioMonitorStageView.java

```diff
@@ -29,8 +35,17 @@
  * Refactored from Android Studio 3.2 adt-ui code.
  */
 public class FlutterStudioMonitorStageView extends FlutterStageView<FlutterStudioMonitorStage> {
+  static final BaseAxisFormatter MEMORY_AXIS_FORMATTER = new MemoryAxisFormatter(1, 5, 5);
+
+  public static final Border MONITOR_BORDER = BorderFactory.createMatteBorder(0, 0, 1, 0, ProfilerColors.MONITOR_BORDER);
+
   private static final int AXIS_SIZE = 100;
 
+  private List<RangedContinuousSeries> rangedData;
+  private LegendComponentModel legendComponentModel;
+  private LegendComponent legendComponent;
+  private Range timeGlobalRangeUs;
```

Comment:
is this selectedTimeRange
?

Reply:
Its suppose to be entire time range collected so I can grab the last item however, it's not quite working - yet.